features/missing.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Missing
+  List missing dependencies
+
+  Scenario: No dependencies
+    Given this Cask file:
+    """
+    """
+    When I run cask "missing"
+    Then I should see command output:
+    """
+    """
+    
+  Scenario: With dependencies
+    Given this Cask file:
+    """
+    (depends-on "package-a" "0.1.2")
+    (depends-on "package-b" "0.2.1")
+    
+    (development
+    (depends-on "package-c")
+    (depends-on "package-d"))
+    """
+    When I run cask "install"
+    And I run cask "missing"
+    Then I should see command output:
+    """
+    """
```

Comment:
This will always be true. It would be better with "Then I should not see command output"